docs/operations-guide/creating-RDS-database-on-AWS.md

```diff
@@ -0,0 +1,98 @@
+![RDS VPC Security Groups](images/RDSVPCSecurityGroup.png)
+
+- Additional configuration
+  - enter `metabase` as the `Initial database name` as this will be the database that Metabase will use for all of its configurations. 
+  - you can also configure the backup window in case you need to restore the backups at some point in time.
+![RDS Initial Database](images/RDSInitialDatabase.png)
+
+When you have completed all these configurations, click on the `Create database` button on the lower right part of the page and wait for the database to be created (this can take several minutes).
+
+## Step 3
+Once the database status is `Available`, you need to click on the DB identifier:
+![RDS DB Identifier](images/RDSDBIdentifier.png)
+
+On the page that appears after you click on the database identifer, you will see on the center of the page the **Connectivity & Security** section. This section will provide you with the **Endpoint** that Metabase needs to connect to the application database we just created.
+
+![RDS Connection Data](images/RDSConnectionData.png)
+
+In the **Security group rules** section, you'll see the Security Group that was created, but the group will have a rule that allows only one IP address to access the database. You need to change this rule to allow access to the Elastic Beanstalk environment.
+![RDS Security Group Rules](images/RDSSecurityGroupRules.png)
+
+To configure the rule for the database, select the INBOUND rule, then click on the Inbound rules tab on the lower part of the page.
+
+![RDS Security Group Rules](images/RDSInboundRule.png)
+
+When you click on Inbound Rules, you need to click on `Edit Inbound Rules` button that appears on the right side of the section
+
+![RDS Edit Inbound Rule](images/RDSEditInboundRule.png)
+
+On the edit page, you need to delete the IP address that appears as default, then add the security group that the Elastic Beanstalk has (the Security group name will have the keyword AWSEBSecurityGroup  in its name). Once you add this security group, click the `Save rules button`.
+
+![RDS Edit Inbound Rule](images/RDSEditInboundRuleSG.png)
+
+# Step 4 
+After having finished all the previous steps, now you can go to the your Elastic Beanstalk deployment and add the RDS instance as the Application Database with [Environment variables](environment-variables.html) under the [Software configuration](running-metabase-on-elastic-beanstalk.html#set-or-change-environment-variables)
+
+---
+# Decouple your RDS database from the Elastic Beanstalk deployment
+
+In the previous versions of this guide, we included the creation and link of a database by default. While this was an easier approach to simplify the deployments, there are many users who can't easily manage the RDS database as it is deeply coupled with the CloudFormation script that was created and executed once they created their Metabase instance.
```

Comment:
This paragraph needs more context around RDS, Beanstalk and Cloudformation. How exactly does decoupling make management easier?